Framework/PythonInterface/plugins/algorithms/CreatePoleFigureTableWorkspace.py

```diff
@@ -79,10 +94,18 @@ def validateInputs(self):
         peak_ws = self.getProperty("PeakParameterWorkspace").value
         chi2_thresh = self.getProperty("Chi2Threshold").value
         x0_thresh = self.getProperty("PeakPositionThreshold").value
+        readout_column = self.getProperty("ReadoutColumn").value
         # if the chi2_threshold is set to allow all peaks, no chi2 column is needed
         self.no_chi2_needed = chi2_thresh < 1e-6
         # if the x0_threshold is set to allow all peaks, no x0 column is needed
         self.no_x0_needed = x0_thresh < 1e-6
+
+        ax_trans = np.asarray(self.getProperty("AxesTransform").value).reshape((3, 3))
+        if np.abs(np.abs(np.linalg.det(ax_trans)) - 1) > 1e-6:
```

Comment:
I sometimes use `np.isclose` to avoid adding magic number tolerances - sometimes a little bit of care is required with the default tolerances but for this case I think it would be OK - unless `1e-6` has some significance?